src/main/resources/jenkins/branch/Messages.properties

```
@@ -28,6 +28,9 @@ DescriptionColumn.displayName=Project description
 ItemColumn.DisplayName=Name
 NamedExceptionsBranchPropertyStrategy.DisplayName=Named branches get different properties
 NoTriggerBranchProperty.suppress_automatic_scm_triggering=Suppress automatic SCM triggering
+NoTriggerBranchProperty.strategy.all=Suppress all, runs will not be triggered by indexing or webhook events
+NoTriggerBranchProperty.strategy.indexing=Suppress runs triggered by indexing
+NoTriggerBranchProperty.strategy.events=Suppress runs triggered by webhook events
```

I find the negative sense here somewhat hard to follow. Usually we prefer to use positive sense in UIs, e.g.
```suggestion
NoTriggerBranchProperty.strategy.all=Suppress builds triggered by indexing or webhook events (only start explicitly)
NoTriggerBranchProperty.strategy.indexing=Suppress builds triggered by indexing (continue to honor webhooks)
NoTriggerBranchProperty.strategy.events=Suppress builds triggered by webhooks (continue to trigger from indexing)
```

Similarly for Groovy in the `properties` step; do you have an updated example of the post-`enum` syntax to review?